htmllize.py

```diff
@@ -57,6 +58,9 @@
       </tr>
 """
 
+WEB_URL_REGEX = r'(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9]\.[^\s]{2,})'
```

Comment:
I am not great on regex, but a comment above this describing what kind URLs you are looking for would be good if someone looks at it later.

Not sure why you use `www`, it could potentially be generic enough to expect any subdomain. 

Another small thing that I can think of is query_string parameters`?key=value&`, although, this is a good first step.